tests/test_api.py

```diff
@@ -30,139 +32,80 @@ def bliss_client(cwd, cfg):
-    )
-    cutout_path = Path(download_dir) / cutout_filename
-    tractor_path = Path(download_dir) / tractor_filename
-    cutout_path.write_bytes(cutout)
-    tractor_path.write_bytes(tractor)
-
-
-def download_rcf_from_git(run, camcol, field, cwd):
-    run_stripped = str(run).lstrip("0")
-    field_stripped = str(field).lstrip("0")
-    run6 = f"{int(run_stripped):06d}"
-    field4 = f"{int(field_stripped):04d}"
-
-    base_url = f"https://api.github.com/repos/prob-ml/bliss/contents/data/sdss/{run}/{camcol}"
-    save_path_rc = Path(cwd) / f"data/sdss/{run}/{camcol}"
-    save_path_rcf = save_path_rc / f"{field}"
-    save_path_rcf.mkdir(parents=True, exist_ok=True)
-
-    assert os.environ["GITHUB_TOKEN"] is not None, "GITHUB_TOKEN environment variable not found"
-    headers = {"Authorization": f"Bearer {os.environ['GITHUB_TOKEN']}"}
-    pf = download_git_lfs_file(f"{base_url}/photoField-{run6}-{camcol}.fits", headers)
-    po = download_git_lfs_file(
-        f"{base_url}/{field}/photoObj-{run6}-{camcol}-{field4}.fits", headers
-    )
-    psf = download_git_lfs_file(
-        f"{base_url}/{field}/psField-{run6}-{camcol}-{field4}.fits", headers
-    )
-    save_file(f"{save_path_rc}/photoField-{run6}-{camcol}.fits", pf)
-    save_file(f"{save_path_rc}/{field}/photoObj-{run6}-{camcol}-{field4}.fits", po)
-    save_file(f"{save_path_rc}/{field}/psField-{run6}-{camcol}-{field4}.fits", psf)
-
-    for band in "ugriz":
-        fpm = download_git_lfs_file(
-            f"{base_url}/{field}/fpM-{run6}-{band}{camcol}-{field4}.fits", headers
+        filename = "sdss_pretrained.pt"
+        bliss_client.load_pretrained_weights_for_survey(
+            survey="sdss",
+            filename=filename,
+            request_headers={"Authorization": f"Bearer {os.environ['GITHUB_TOKEN']}"},
         )
-        frame = download_git_lfs_file(
-            f"{base_url}/{field}/frame-{band}-{run6}-{camcol}-{field4}.fits", headers
+        not_found_err_msg = (
+            "pretrained weights "
+            + f"{bliss_client.cwd}/data/pretrained_models/{filename} "
+            + "not found"
+        )
+        assert Path(
+            bliss_client.cwd + f"/data/pretrained_models/{filename}"
+        ).exists(), not_found_err_msg
+
+    def test_train(self, bliss_client, monkeypatch):
+        monkeypatch.setattr(api, "_train", mock_train)
+        bliss_client.train("test", training={"n_epochs": 1})
+
+    def test_load_survey(self, bliss_client, monkeypatch):
+        monkeypatch.setattr(requests, "get", mock_get)
+        monkeypatch.setattr(requests, "post", mock_post)
+        monkeypatch.setattr(gzip, "decompress", lambda x: x)
+        monkeypatch.setattr(bz2, "decompress", lambda x: x)
+
+        download_dir = "data"
+        bliss_client.load_survey("sdss", 94, 1, 12, download_dir)
+        assert os.path.exists(f"{bliss_client.cwd}/{download_dir}/94/1/photoField-000094-1.fits")
```

Comment:
`load_survey` "downloads" the mocked data and saves it to the files. So they exist, they just contain garbage, but since we don't need the data in this test case that doesn't matter.

Reply:
Ah got it, makes sense. Thanks.